test/e2e/volume_provisioning.go

```diff
@@ -184,6 +186,70 @@ var _ = framework.KubeDescribe("Dynamic provisioning", func() {
 			Expect(err).To(HaveOccurred())
 			framework.Logf(err.Error())
 		})
+
+		It("should test that deleting a claim before the volume is provisioned deletes the volume. [Volume]", func() {
```

Comment:
L204-205: you want an `Excpect` for the `StorageClasses.Create` call and for the `StorageClasses().Delete` call. The `defer` should probably be a func() {...}